### PR TITLE
[10.6.0] Rename Phoenix to Web

### PR DESCRIPTION
**Pull request by phil-davis**
Port of #38199 to release-10.6.0

### CODE REVIEW

**Comment by phil-davis** [comment]
@micbar please advise what should happen with the changelog entry.
Maybe you will run some changelog-rebuild thing after this is merged to the release branch, or?

**Comment by phil-davis** [comment]
Note: the SonarCloud negative report is because of PRs like #38147 in `master` that improved some "bugs". This "backport" branch sees code that is not as "good" as the code in master.

The "required" codecov results need to be ignored. We now use SonarCloud. I am not sure why GitHub thinks that codecov is required any more.

@micbar

File: lib/private/NavigationManager.php [hunk 1]
```diff
@@ -190,15 +190,24 @@ private function init() {
...
+		$webIconLabel = 'web.label';
+		if (!$webBaseUrl) {
+			// Check the old phoenix.baseUrl system key to provide compatibility across
+			// the name change from phoenix to web.
+			$webBaseUrl = \OC::$server->getConfig()->getSystemValue('phoenix.baseUrl', null);
+			$webIconKey = 'phoenix.icon';
+			$webIconLabel = 'phoenix.label';
+		}
+		if ($webBaseUrl) {
+			$iconPath = $this->config->getSystemValue($webIconKey, $this->urlGenerator->imagePath('core', 'apps/phoenix.svg'));
```
Comment (jnweiger) [inline lib/private/NavigationManager.php:205]: Is it correct to use 'core/img/apps/phoenix.svg` here? Do we want to rename to web.svg?

**Review by jnweiger — APPROVED** [review]
it still refrences core/img/phoenix.svg -- should we rename that svg too?

File: lib/private/NavigationManager.php (hunk 1 above)
Comment (phil-davis) [reply lib/private/NavigationManager.php:205]: Fixed with PR #38201 and #38202

**Pull request merged**